src/page/main-page.tsx

```diff
@@ -120,12 +121,20 @@ export const MainPage: FunctionComponent = () => {
                 );
               })}
           </div>
+
           {documentStoreStatus && activeFeatureBlock === "issue-revoke" && activeSubFeatureBlock === "revoke" && (
-            <RevokeBlock documentStoreAddress={documentStoreAddress} />
+            <>
+              <RevokeBlock documentStoreAddress={documentStoreAddress} />
+              <RevokePanel />
+            </>
           )}
           {documentStoreStatus && activeFeatureBlock === "issue-revoke" && activeSubFeatureBlock === "issue" && (
-            <IssueBlock documentStoreAddress={documentStoreAddress} />
+            <>
+              <IssueBlock documentStoreAddress={documentStoreAddress} />
+              <IssuePanel />
```

Comment:
It's not very obvious the panel change when we set the document store, so I don't really like it as is.

Idea: maybe add question marks and open panels when a user click on it?

Also we should work on the panel design, it's very raw